.github/workflows/coverage.yml

```diff
@@ -33,6 +33,10 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}
 
+    # TODO(Crissman): Remove click after issue with allennlp import resolved
+    - name: Install click 7.1.2 until allennlp import issue resolved
+      run: pip install --use-deprecated=legacy-resolver --progress-bar off click==7.1.2
```

Comment:
A nit, but how about placing this step after `Setup environment`?
I think adjacent pip installation steps will improve the readability.